internal/api/client.go

```diff
@@ -78,14 +78,31 @@ func (c *Client) CreateCluster(request *CreateClusterRequest) (*model.Cluster, e
 	defer closeBody(resp)
 
 	switch resp.StatusCode {
-	case http.StatusOK:
+	case http.StatusAccepted:
```

Comment:
All modification endpoints now return accepted instead of ok, since the actual changes are of course still pending.